trino-s3-proxy/src/test/java/io/trino/s3/proxy/server/LocalServer.java

```diff
@@ -31,7 +31,10 @@ private LocalServer() {}
     public static void main(String[] args)
     {
         TestingTrinoS3ProxyServer trinoS3ProxyServer = TestingTrinoS3ProxyServer.builder()
-                .withMockS3Container()
+                .withS3Container()
+                .withPostgresContainer()
+                .withMetastoreContainer()
+                .withPySparkContainer()
```

Comment:
Do we normally want all of these containers as part of a standard server execution?

Reply:
I'm not actually sure tbh. I think, actually, that `LocalServer` will go away in favor of some kind of CLI or Docker container with options so I think this is fine for now.